candi/bashible/common-steps/node-group/063_install_kubelet_forker.sh.tpl

```diff
@@ -21,6 +21,8 @@ if [ -x /opt/deckhouse/bin/sysctl-tuner ]; then
   /opt/deckhouse/bin/sysctl-tuner
 fi
 
+sysctl -w kernel.panic=10
```

Comment:
Comment is required with the info why we are doing this

Reply:
Fixed